### PR TITLE
added dir brute force tools

### PR DESCRIPTION
**Pull request by yaman-694**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,11 @@ Check the contribution [RULES](/CONTRIBUTING.md)
     - [Web Cache Poisoning](/Attacks/README.md)
     - [XSS](/Attacks/README.md)
 
+- [Directory Bruteforce tools](/Directory%20Bruteforce%20tools/README.md)
```
Comment (Prashant-Bhapkar) [inline README.md:68]: Check alphabetical order

File: README.md [hunk 2]
```diff
@@ -65,6 +65,11 @@ Check the contribution [RULES](/CONTRIBUTING.md)
...
+    - [Gobuster](/Directory%20Bruteforce%20tools/README.md)
```
Comment (Prashant-Bhapkar) [inline README.md:69]: Check alphabetical order

File: Directory Bruteforce tools/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+## Directory Bruteforce tools
```
Comment (Prashant-Bhapkar) [inline Directory Bruteforce tools/README.md:1]: Make "T for tools" should be for the folder name.

**Comment by yaman-694** [comment]
Changes done

**Comment by Prashant-Bhapkar** [comment]
Please give a "Starred⭐" to this repository if you like this repo.

**Pull request merged**